### PR TITLE
Bump node-gyp version

### PR DESCRIPTION
**Pull request by hubertdeng123**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6807,22 +6950,23 @@ fsevents@~2.1.2:
   linkType: hard
 
 "node-gyp@npm:latest":
-  version: 7.1.2
-  resolution: "node-gyp@npm:7.1.2"
+  version: 9.4.0
```
Comment (hubertdeng123) [inline yarn.lock:6953]: look here

Comment (chadwhitacre) [reply yarn.lock:6953]: Needle in a haystack!

Comment (chadwhitacre) [reply yarn.lock:6953]: There's no way to keep the diff smaller?

**Review by mattgauntseo-sentry — APPROVED** [review]
LGTM - It seems like fsevents wants node-gyp latest, so this should be safe enough.

File: yarn.lock (hunk 1 above)
Comment (hubertdeng123) [reply yarn.lock:6953]: I don't think so :(

I think this should be safe enough though

**Pull request merged**